devito/ir/clusters/algorithms.py

```diff
@@ -6,12 +6,13 @@
 from devito.ir.support import Any, Backward, Forward, IterationSpace, Scope
 from devito.ir.clusters.cluster import Cluster, ClusterGroup
 from devito.symbolics import CondEq, xreplace_indices
-from devito.tools import DAG, as_tuple, flatten, filter_ordered, generator
+from devito.tools import DAG, as_tuple, flatten, filter_ordered, generator, timed_pass
```

Comment:
timed_pass? wouldn't `time_passed` be better?

Reply:
"pass" stands for "compiler pass". So it's a "timed compiler pass". It's pretty standard compiler jargon